src/bun.js/node/node_fs.zig

```diff
@@ -3013,12 +3013,15 @@ pub const NodeFS = struct {
 
     pub fn mkdtemp(this: *NodeFS, args: Arguments.MkdirTemp, comptime flavor: Flavor) Maybe(Return.Mkdtemp) {
         var prefix_buf = &this.sync_error_buf;
-        prefix_buf[0] = 0;
         const len = args.prefix.len;
         if (len > 0) {
             @memcpy(prefix_buf, args.prefix.ptr, len);
-            prefix_buf[len] = 0;
         }
+        var i: usize = 0;
+        while (i < 6) : (i += 1) {
+            prefix_buf[len + i] = 'X';
```

Comment:
```suggestion
            prefix_buf[len..][0..6].* = "XXXXXX"
            prefix_buf[len..][6] = 0
```

Reply:
We can remove the loop and this code will be slightly faster most likely

Reply:
fixed. didn't know zig has such syntax before :p